R/convert-from-datacamp.R

```diff
@@ -16,7 +16,7 @@
 #' datacamp_to_learnr_pkg(lrnr_pkg_path = Temp)
 #' }
 datacamp_to_learnr_pkg <-
-  function(dc_path = ".", lrnr_pkg_path,
+  function(dc_path = ".", lrnr_pkg_path = ".",
```

Comment:
I'd recommend the default lrnr pkg path be not set, since we don't want to convert into a package in the same repo as per #2